ckanext/switzerland/templates/emails/subscribe_deletion_plain_text.txt

```diff
@@ -14,14 +14,14 @@ Freundliche Grüsse
 Die Geschäftsstelle OGD
 
 Bonjour
-Nous vous informons que le jeu de données « {{ notifications[0].object_title_fr }} » auquel vous vous êtes abonné-e a été retiré de notre portail par le fournisseur de données.
+Nous vous informons que le jeu de données «{{ notifications[0].object_title_fr }}» auquel vous vous êtes abonné-e a été retiré de notre portail par le fournisseur de données.
```

Comment:
why this quotes looks one time « and another "

Reply:
I wondered the same thing for a second 😇 But it is the template we got and in french you usually use these kind of quotes, but thanks for checking!